src/controllers/mainController.ts

```diff
@@ -1525,11 +1548,13 @@ export default class MainController implements vscode.Disposable {
             if (editor.document.getText(selectionToTrim).trim().length === 0) {
                 return;
             }
+
             await self._outputContentProvider.runQuery(
                 self._statusview,
                 uri,
                 querySelection,
                 title,
+                self._executionPlanOptions,
             );
         } catch (err) {
             console.warn(`Unexpected error running query : ${err}`);
```

Comment:
I know it's not your change, but can you change `${err}` to `${getErrorMessage(err)}`?